eclair-core/src/test/scala/fr/acinq/eclair/StartupSpec.scala

```diff
@@ -82,6 +82,26 @@ class StartupSpec extends AnyFunSuite {
     assert(nodeParamsAttempt.failed.get.getMessage == "requirement failed: configuration key 'features' have moved from bytevector to human readable (ex: 'feature-name' = optional/mandatory)")
   }
 
+  test("NodeParams should not merge user defined features with default features") {
```

Comment:
Testing should instead be done in eclair-node, and it would check that only features that are defined in its `application.conf` file are actually activated. This is not the case now and a failing test could be a good first commit for this PR.